open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-basics.ipynb

```diff
@@ -1,12 +1,27 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    d = my_dict.get('d')
</pre><p>recover the code if you don't want to commit them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='102'/>